azure-pipelines.yml

```diff
@@ -68,6 +68,16 @@ steps:
       Arguments: 'bundleAnalyze:collect'
     displayName: 'Run bundle analysis and collect'
 
+  - script: 'node prepNewReadme.js'
+    workingDirectory: '$(System.DefaultWorkingDirectory)\packages\teams-js'
+    condition: and(
+      eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'),
```

Comment:
I think we only want to update the README.md when the integrity hash is being generated for the MicrosoftTeams.min.js file that is going to be pushed to prod CDN. We always want the HTML script tag that has a prod CDN URL to include the integrity hash that is valid for it. 

If I understand the trigger correctly, I think this will execute on every CI build to 2.0-preview. 

This pipeline still reflects all the steps that are done to publish as we haven't yet onboarded the release pipeline that will take care of those steps.

The matching condition that I think we want here (given that it's still reflecting our current/soon-to-be-old release process) is:

`startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),`

Does this make sense?

Reply:
Yes, this makes perfect sense. Since we publish next-dev versions after merges (2.0.0-beta.x-dev.x), do we care about those dev versions Readmes on npm being correct?

If so, I think we might want both triggers, like the triggers set for the publishing to npm step?
```
or(startsWith(variables['Build.SourceBranch'], 'refs/tags/v'), eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview')),
```

Reply:
The README file on @next-dev packages published to npm will still just refer to prod CDN, so there's no need to update the integrity check. I think. Unless we want to update those with the SDF CDN URL rather than the prod CDN URL. I don't think we've considered modifying the README to point to SDF CDN for those @next-dev releases. We should probably discuss this further offline.

Reply:
Fixed